### ISSUE TITLE
leo `attachment` icon being selected twice when tabbing

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#47865]
### Description

Quick follow up of https://github.com/brave/brave-browser/issues/47796. When tabbing through the `Leo` side panel, the `attachment` icon is being selected twice. Users shouldn't need to press `tab` twice to select the `attachment` icon.

### Steps to reproduce

1. install `Brave` and load any website within a tab
2. open `Leo` via the side panel and the `input/textbox` should be focused by default
3. start tabbing through the `Leo` elements and notice that you need to tab through the `attachment` icon twice

### Actual result

https://github.com/user-attachments/assets/284baa31-b02b-4f83-8092-cffcb2b7a0d3

### Expected result

Users shouldn't need to press `tab` twice to select the `attachment` icon.

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

```
Brave | 1.82.106 Chromium: 139.0.7258.42 (Official Build) nightly (64-bit)
-- | --
Revision | ac7898849899c1331a3319c7a6e9ef766ccb9b26
OS | Windows 11 Version 24H2 (Build 26100.4484)
```

### Channel information

- [x] release (stable)
- [x] beta
- [x] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome
